src/logic/basic.lean

```diff
@@ -59,7 +59,7 @@ instance psum.inhabited_right {α β} [inhabited β] : inhabited (psum α β) :=
   {α} [subsingleton α] : decidable_eq α
 | a b := is_true (subsingleton.elim a b)
 
-@[simp] lemma eq_iff_true_of_subsingleton [subsingleton α] (x y : α) :
+@[simp] lemma eq_iff_true_of_subsingleton {α : Sort*} [subsingleton α] (x y : α) :
```

Comment:
It wouldn't surprise me if this breaks something - since you no longer need it (and I just opened #7061), I recommend you back this change out.

Reply:
Ok, I went back to `Type*` in `eq_iff_true_of_subsingleton`.